src/averageAges.js

```diff
@@ -15,7 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  let men;
+
+  century
+    ? men = people.filter(person => person.sex === 'm'
```

Comment:
Part of this line has code that is repeated. Could you possibly turn it into a variable or a global variable?